internal/metrics/metrics.go

```diff
@@ -14,20 +14,18 @@ import (
 var TotalSentAvailabilityCheckReqs = prometheus.NewCounterVec(
 	prometheus.CounterOpts{
 		Name:        "provisioning_source_availability_check_request_total",
-		Help:        "availability check requests count partitioned by type (aws/gcp/azure), source status, component and error",
+		Help:        "availability check requests processed successfully or not partitioned by type (aws/gcp/azure), source status, component and error",
 		ConstLabels: prometheus.Labels{"service": version.PrometheusLabelName, "component": "statuser"},
 	},
 	[]string{"type", "status", "error"},
 )
 
-var TotalReceivedAvailabilityCheckReqs = prometheus.NewCounterVec(
+var TotalEnqueueAvailabilityCheckReqs = prometheus.NewCounter(
```

Comment:
Could you pls explain to us noobs what is a difference between `CounterVec` and `Counter`? :pleading_face:

Reply:
just the labels, I didn't think of helpful labels to use in this new metric so we need only one counter and not a vector of counters for each label. 

`CounterVec is a Collector that bundles a set of Counters that all share the same Desc, but have different values for their variable labels.`

Reply:
Ah! right, make sense! Thanks for explaining :)